### PR TITLE
SW-4906 Move the documents list to view-agnostic, implement in participant and accelerator views

### PR DESCRIPTION
**Pull request by nickgraz**

### CODE REVIEW

**Comment by nickgraz** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2214?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

Current dependencies on/for this PR:
* **#2214** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2214?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#2212** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2212?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/2214?utm_source=stack-comment).

File: src/components/DeliverableView/index.tsx [hunk 1]
```diff
@@ -36,7 +40,7 @@ const DeliverableView = (props: Props): JSX.Element => {
             <StatusBar {...viewProps} />
             <Description {...viewProps} />
             <DocumentsUploader {...viewProps} />
-            <DocumentsList {...viewProps} />
+            {isAcceleratorRoute ? <AcceleratorDocumentsList {...props} /> : <DocumentsList {...props} />}
```
[the remark this answers is not in the document]
Comment (karthikbtf) [reply src/components/DeliverableView/index.tsx:43]: Having view specific components is my preference. That's how I would have done it.
I misinterpreted what having a 'common view' was from last week's meeting.
Yes, let's assemble views specific to each scene and just pull in shared components. 
Thanks 🙏

[the remark this answers is not in the document]
Comment (nickgraz) [reply src/components/DeliverableView/index.tsx:43]: Cool I will do that in the next PR

[the remark this answers is not in the document]
Comment (nickgraz) [reply src/components/DeliverableView/index.tsx:43]: #2215

**Comment by nickgraz** [comment]
### Merge activity

* **Feb 27, 4:27 PM EST**: @nickgraz started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/2214).

**Pull request merged**